src/lib/api.tsx

```diff
@@ -11,13 +11,22 @@ import {
   AdditionalHeadersType
 } from "../types";
 
-const baseURL = "https://discussion.theguardian.com/discussion-api";
-
-let additionalHeaders = {};
+let options = {
+  // Defaults
+  baseUrl: "https://discussion.theguardian.com/discussion-api",
```

Comment:
I've set this to default to production but in reality the prop is required so this should never be needed